common/djangoapps/student/migrations/0049_auto__add_languageproficiency__add_unique_languageproficiency_code_use.py

```diff
@@ -173,7 +173,7 @@ def backwards(self, orm):
             'courseware': ('django.db.models.fields.CharField', [], {'default': "'course.xml'", 'max_length': '255', 'blank': 'True'}),
             'gender': ('django.db.models.fields.CharField', [], {'db_index': 'True', 'max_length': '6', 'null': 'True', 'blank': 'True'}),
             'goals': ('django.db.models.fields.TextField', [], {'null': 'True', 'blank': 'True'}),
-            'has_profile_image': ('django.db.models.fields.BooleanField', [], {'default': 'False'}),
+            'profile_image_version': ('django.db.models.fields.CharField', [], {'max_length': '16', 'null': 'True'}),
```

Comment:
I'm not sure I understand the rationale of keeping this as a string. Are you thinking that we might want a different versioning scheme in the future? I would prefer to have a field called 'profile_image_last_updated' that is an int field. I don't know enough about our scaling issues to know whether it is important to keep these fields small, but with 3 million user I think it would be better to have a four byte field rather than sixteen bytes.

This is just my not very strongly held opinion, so I'm happy to go with the code as is if that's what everyone is comfortable with.